udata/core/dataset/rdf.py

```diff
@@ -678,7 +683,11 @@ def dataset_from_rdf(graph: Graph, dataset=None, node=None):
     identifier = rdf_value(d, DCT.identifier)
     uri = d.identifier.toPython() if isinstance(d.identifier, URIRef) else None
     remote_url = remote_url_from_rdf(d)
-    created_at = rdf_value(d, DCT.issued)
+    created_at = rdf_value(d, DCT.created)
```

Comment:
`DCT.created` [comes from GeoDCAT-AP](https://semiceu.github.io/GeoDCAT-AP/releases/2.0.0/) only, if you wanna add a comment